docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -168,37 +168,212 @@ Make up your mind on which approach you want to use.
+quarkus.hibernate-orm."inventory".packages=org.acme.model.inventory
+----
+<1> Define a datasource named `users`.
+<2> Define a datasource named `inventory`.
+<3> Define a persistence unit called `users`.
+<4> Define the datasource used by the persistence unit.
+<5> This configuration property is important but we will discuss it a bit later.
+<6> Define a persistence unit called `inventory` pointing to the `inventory` datasource.
+
+[NOTE]
+====
+You can mix the default datasource and named datasources or only have one or the other.
+====
+
+[NOTE]
+====
+The default persistence unit points to the default datasource by default.
+For named persistence units, the `datasource` property is mandatory.
+You can point your persistence unit to the default datasource by setting it to `<default>`
+(which is the internal name of the default datasource).
+
+It is perfectly valid to have several persistence units pointing to the same datasource.
+====
+
+==== Attaching model classes to persistence units
+
+There are two distinct ways to attach model classes to persistence units and they should not be mixed:
+
+* Via the `packages` configuration property;
+* Via the `@io.quarkus.hibernate.orm.PersistenceUnit` package-level annotation.
+
+If both are mixed, the annotations are ignored and only the `packages` configuration properties are taken into account.
+
+Using the `packages` configuration property is simple:
+
+[source,properties]
+----
+quarkus.hibernate-orm.database.generation=drop-and-create
+quarkus.hibernate-orm.packages=org.acme.model.defaultpu
+
+quarkus.hibernate-orm."users".database.generation=drop-and-create
+quarkus.hibernate-orm."users".datasource=users
+quarkus.hibernate-orm."users".packages=org.acme.model.user
+----
+
+This configuration snippet will create two persistence units:
+
+* The default one which will contain all the model classes under the `org.acme.model.defaultpu` package, subpackages included.
+* A named `users` persistence unit which will contain all the model classes under the `org.acme.model.user` package, subpackages included.
+
+It is possible to attach several packages to a persistence unit:
+
+[source,properties]
+----
+quarkus.hibernate-orm."users".packages=org.acme.model.shared,org.acme.model.user
+----
+
+All the model classes under the `org.acme.model.shared` and `org.acme.model.user` packages will be attached to the `users` persistence unit.
+
+It is also supported to attach a given model class to several persistence units.
+
+[NOTE]
+====
+Model classes need to be consistently added to a given persistence unit.
+Meaning all dependent model classes of a given entity (mapped super classes, embeddables...) are required to be attached to the persistence unit.
+As we are dealing with the persistence unit at the package level, it should be simple enough.
+====
+
+[WARNING]
+====
+Panache entities can be attached to only one persistence unit.
+
+For entities attached to several persistence units, you cannot use Panache.
```

Comment:
This is confusing. [Here](https://github.com/quarkusio/quarkus/pull/11826/files#diff-47831eaea1f7b8c85191d0fb3e7e96dcR283 ) we propose how to attach an entity to several PUs but now we're saying to not use Panache in case that happens?

Reply:
Yes. Thus this sentence.

Reply:
Then I think the sentence `It is also supported to attach a given model class to several persistence units` needs to be removed?

Reply:
Ah you're referring to Hibernate, not Panache, now I get it

Reply:
Yes, exactly :).